scene/3d/path_3d.cpp

```diff
@@ -431,6 +435,7 @@ bool PathFollow3D::is_using_model_front() const {
 
 void PathFollow3D::set_loop(bool p_loop) {
 	loop = p_loop;
+	_update_transform();
```

Comment:
Will all these `_update_transform()` calls lead to doing multiple chained updates unnecessarily, e.g. when setting all the properties when constructing the object from a scene file?

Should this be cached with a dirty update pattern, or deferred?

Reply:
> Will all these _update_transform() calls lead to doing multiple chained updates unnecessarily

I am not sure, I just followed the existing convention, such as
```
void PathFollow3D::set_v_offset(real_t p_v_offset) {
	v_offset = p_v_offset;
	if (path) {
		_update_transform();
	}
}
```

> Should this be cached with a dirty update pattern, or deferred?

I am not sure it is worthwhile for two reasons
- `_update_transform` is not that expensive nor frequently called
- to defer the call we need to bind `_update_transform`, which should be kept away from end user.

Of course, I am not that familiar with the code base. If you think it need to be cached I can do it.

Reply:
> cached with a dirty update pattern

done